src/translations/english.js

```diff
@@ -0,0 +1,28 @@
+export default {
```

Comment:
this might be a good place to introduce typescript or something similar that would check for missing keys in the object. testing for all languages will become very difficult

Reply:
will take this up separately https://github.com/covid-maps/covid-maps/issues/162